native/templates/openharmony/entry/src/main/ets/MainAbility/workers/cocos_worker.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2021 Huawei Device Co., Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import worker from '@ohos.worker';
+import nativerender from "libcocos2d.so";
+import { ContextType } from "../common/Constants"
+import { log } from '../cocos/log_utils';
+import { launchEngine } from '../cocos/game'
+
+console.log("cocos worker: New Worker Thread")
+
+const nativeContext = nativerender.getContext(ContextType.WORKER_INIT);
+nativeContext.workerInit()
+
+const parentPort = worker.parentPort;
+
+var renderContext: any = undefined;
+parentPort.onmessage = function(e) {
+    var data = e.data;
+
+    console.log("kee cocos worker type:" + data.type);
+    switch(data.type) {
+        case "AppLifecycle":
+            console.log("kee cocos worker: AppLifecycle");
+            const nativeAppLifecycle = nativerender.getContext(ContextType.APP_LIFECYCLE);
+            switch(data.data) {
+                case "onCreate":
+                    console.log("kee cocos worker: onCreate");
```

Comment:
this is debug
```suggestion
                    console.log("cocos worker: onCreate");
```

this is debug log too